mappings/net/minecraft/world/entity/EntityTrackingGroup.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/class_5572 net/minecraft/world/entity/EntityTrackingGroup
```

Comment:
Group is a very loose description for this class, given its usage to specifically represent a section containing entities. It also holds state relating to the chunk that contains it.

Reply:
I like to think of it as a "group" of entities that are processed together by the manager; is there an established naming scheme for these already?

Reply:
By their usage, they are used to represent chunk sections for entities. Especially given they reference chunk state it makes sense to name according to that.

Reply:
> It also holds state relating to the chunk that contains it.

it occurs to me that the problem name here might actually be `EntityChunkStatus`, which is actually a flags enum encoding whether the entities in this group should be ticked, tracked, or neither. the "state" that you mention isn't related to the parent chunk, it's exclusively related to the group

Reply:
The issue for me comes by reading the name "Group", I can have no idea along what lines entities are going to be grouped. It seems totally arbitrary- and hence any code working with these becomes unclear. You will always be working with a "section" of entities through this class.

Reply:
OK - sounds good to me. `EntityTrackingSection` then?